docs/source/configuration.rst

```diff
@@ -1,44 +1,72 @@
 Configuration
 =============
 
+UCX can be configured with a wide variety of options and optimizations including: transport, caching, etc.
+
+For novice user we recommend the following settings:
+
+::
+
+    UCX_MEMTYPE_CACHE=n UCX_TLS=all
```

Comment:
Should we add an explanation of `UCX_MEMTYPE_CACHE` below as well?

Reply:
There's an explanation of UCX_MEMTYPE_CACHE later in configuration docs